internal/venice-common/src/main/java/com/linkedin/venice/pushstatushelper/PushStatusStoreReader.java

```diff
@@ -101,7 +101,9 @@ public Map<CharSequence, Integer> getPartitionStatus(
       if (pushStatusValue == null) {
         return Collections.emptyMap();
       } else {
-        LOGGER.info(" {}/{} Instance status: {}", storeName, partitionId, pushStatusValue.instances);
+        if (pushStatusValue.instances.size() < 50) {
+          LOGGER.info(" {}/{} Instance status: {}", storeName, partitionId, pushStatusValue.instances);
```

Comment:
Can we  log store version as well? Using

```java
LOGGER.info(" {}_v{}-{} Instance status: {}", storeName, version, partitionId, pushStatusValue.instances);
```

Reply:
Can we log the first 50 `incomplete` instances?
`completed` instances are normally not useful for debugging, and we mostly care about `incomplete` instances.

Reply:
Also it is good to provide a summary about the instance status, such as the count for different instance status and maybe log several instances for the `incomplete` status.
@sixpluszero 
Do you have any idea about how to summarize the DVC push status from the debugging POV?

Reply:
I think the return statuses are logged in `getDaVinciPartitionLevelPushStatusAndDetails` so it should be fine to skip logging it here.